pkg/variantregistry/ocp.go

```diff
@@ -443,6 +443,7 @@ func setSuite(_ logrus.FieldLogger, variants map[string]string, jobName string)
 		{"-etcd-scaling", "etcd-scaling"},
 		{"conformance", "parallel"}, // Jobs with "conformance" but no explicit serial are probably parallel
 		{"usernamespace", "usernamespace"},
+		{"machine-config-operator", "machine-config-operator"},
```

Comment:
I would use Procedure variant here, we're looking to remove Suite. It can't be reliably detected.